bin/dynamic

```diff
@@ -14,8 +14,8 @@
 #                                                                      #
 ########################################################################
 
-# Increment this if and only if binary incompatibility is introduced
-DYLIB_VERSION=93u+m.1
+# Increment major version if and only if binary incompatibility is introduced
+AST_DYLIB_VERSION=2.0
```

Comment:
> Does anyone know what the highest library version number used by AT&T was? We should go one higher than that.

These are the library versions in ksh93v- (note that to get dynamic libraries to build with `nmake` the `-nostartfiles` flag must be deleted from `src/cmd/INIT/make.probe`):
```sh
$ ls arch/linux.i386-64/lib/*.so.*
arch/linux.i386-64/lib/libast.so.5.4  arch/linux.i386-64/lib/libcmd.so.1.2  arch/linux.i386-64/lib/libdll.so.1.0  arch/linux.i386-64/lib/libshell.so.1.1
```

Reply:
Thanks. So they all have different versions. My script gives them all the same version number, which I think only makes sense as they all depend on the same libast. So I'll up that to 6.0.